tools/functional-tester/etcd-runner/command/lock_racer_command.go

```diff
@@ -61,7 +64,7 @@ func runRacerFunc(cmd *cobra.Command, args []string) {
 				break
 			}
 		}
-		m := concurrency.NewMutex(s, "racers")
+		m := concurrency.NewMutex(s, racers)
```

Comment:
same as election code-- if there's a conflict I don't think using a random string really solves the problem